adminapp/src/pages/PaymentTriggerForm.jsx

```diff
@@ -91,15 +95,15 @@ export default function PaymentTriggerForm({
           This memo is used for all triggered subsidy transactions.
         </FormHelperText>
         <AutocompleteSearch
-          {...register("originatingLedger")}
+          {...register("originatingLedgerId")}
```

Comment:
Everywhere else (like book transaction create) this uses the reference, not an ID, what's up here?

Reply:
Elsewhere we register as originatingLedger but when we call the update API, we use their `.id`. Also in BookTransactionsCreate, we are not using the new common resource edit/create reusable components system. Wondering if we should rename the book transactions to originatingLedgerId or leave as is.